### ISSUE TITLE
Opening vscode via vscode-recent makes vscode 'forget' node on commit

### ISSUE THREAD

**Issue by WandersonAlves** [plibither8/ulauncher-vscode-recent#9]
When making a commit using vscode Source Control tab, vscode complains about node

![image](https://user-images.githubusercontent.com/9141449/138333493-46918d89-d3fc-47c9-9d32-a117c6c0053d.png)

The error:
```
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -
Yarn requires Node.js 4.0 or higher to be installed.
husky - pre-commit hook exited with code 1 (error)
```

Using the internal terminal works. Only on Source Control tab this problem occurs.

Some info:
```sh
➜ which code    
/usr/bin/code
```

**Comment by mattalexx** [comment]
@WandersonAlves This seems like a problem with VS Code. Are you saying that the problem only occurs when you use Ulauncher to open the file?

**Comment by WandersonAlves** [comment]
Yes, if open my projects in the old way (`cd` into folder and `code .`) source control works as expected. If a open via ulauncher, vscode complains about node (only on source code tab). Internal terminal works as expected in both scenarios
